src/python/pants/engine/internals/graph.py

```diff
@@ -334,7 +338,9 @@ async def resolve_target(
     parametrizations = await Get(
         _TargetParametrizations,
         _TargetParametrizationsRequest(
-            base_address, description_of_origin=request.description_of_origin
+            base_address,
+            description_of_origin=request.description_of_origin,
+            ignore_unrecognized_build_file_symbols=request.ignore_unrecognized_build_file_symbols,
```

Comment:
Since the use case for this is for the entire run of the bootstrap scheduler, it seems like maybe it could be set as a global or `SessionValue` to avoid needing to change the public API at all?

Reply:
Discovered over DM. I tried to surface `PySession.build_id` to look for the session `plugin_resolver`, but I realized it didn't work because the relevant rule can't request the `PySession`. We also discussed that `SesssionValues` is bad because of unnecessary cache invalidation.

I proposed that this comment isn't that big of a deal. It's a bummer the API creeped into 3-4 call sites, but those are all dead simple and the default arg means most people can ignore this.